melodyPortfolio.html

```diff
@@ -3,34 +3,132 @@
+									</p>
+ 									<div class="project_links_bar">	
+ 										<a href="">Github</a>
+ 										<a href="">Live</a>
+									</div>
+							</div>	
+			
+							<div class="project_card">
+									<img src="images/random1.jpg" alt="An illustration of this project">
+									<h3>Project title</h3>
+									<p>A short and impactful description of the project. A nice little paragraph where I talk about the project. What was it about? What techniques did I use? What did I enjoy 										about it? Why do I think it is interesting? Any highlights, fun facts, interesting results?
+									</p>
+ 									<div class="project_links_bar">	
+ 										<a href="">Github</a>
+ 										<a href="">Live</a>
+									</div>
+							</div>
+
+							<div class="project_card">
+									<img src="images/random2.jpg" alt="An illustration of this project">
+									<h3>Project title</h3>
+									<p>A short and impactful description of the project. A nice little paragraph where I talk about the project. What was it about? What techniques did I use? What did I enjoy 											about it? Why do I think it is interesting? Any highlights, fun facts, interesting results?
+									</p>
+									<div class="project_links_bar">	
+											<a href="">Github</a>
+											<a href="">Live</a>
+									</div>
+							</div>	
+			
+							<div class="project_card">
+									<img src="images/random3.jpg" alt="An illustration of this project">
+									<h3>Project title</h3>
+									<p>A short and impactful description of the project. A nice little paragraph where I talk about the project. What was it about? What techniques did I use? What did I enjoy 											about it? Why do I think it is interesting? Any highlights, fun facts, interesting results? This is an additional sentence to test what happend when tthis paragraph 											has a slightly different size. Hope everything stays in place!
+									</p>
+ 									<div class="project_links_bar">	
+ 										<a href="">Github</a>
+ 										<a href="">Live</a>
+									</div>
+							</div>	
+			
+							<div class="project_card">
+									<img src="images/random1.jpg" alt="An illustration of this project">
+									<h3>Project title</h3>
+									<p>A short and impactful description of the project. A nice little paragraph where I talk about the project. What was it about? What techniques did I use? What did I enjoy 											about it? Why do I think it is interesting? Any highlights, fun facts, interesting results? 
+									</p>
+ 									<div class="project_links_bar">	
+ 										<a href="">Github</a>
+ 										<a href="">Live</a>
```

Comment:
for empty links, it is good to put `href="#"`, because that way it behaves more like a working link. just best practice

Reply:
done